### PR TITLE
Makeshift Battleaxe

### PR DESCRIPTION
**Pull request by assortedbeads**
## About The Pull Request

Adds a makeshift battle axe to round out the makeshift weapon series.

More damage then the hatchet but less than the fireaxe
More pen than the hatchet but less than the fire axe
Less breaching than the fire axe but slightly more than the junkblade
Has better cutting than the hatchet

Has a broad strike and a swing. Like the junkblade and mace it doesn't have double tac

Stats compared to the junkblade

Axe StatsJunkblade Stats

Sprites (They're mostly just palette swapped fireaxe sprites with minimal differences)

Wield Back Wield Front
Wield Side 2 Wield Side

Single Back Single Front
Single Side 2 Single Side

Dual Back Dual Front
Dual Side 2 Dual Side

**Note:**
This PR also fixes the pixel issue in
https://github.com/discordia-space/CEV-Eris/pull/7935
While this isn't "atomized" it's the same file being affected, and I really don't want to have to redo this because of two pixels.

## Why It's Good For The Game

You can now have a makeshift version of each weapon type; Axe, Hammer, Sword.

I personally wanted a scrap axe and the halberd doesn't feel like that for me.

It's also a good decent weapon against tankier enemies.

## Testing

- Spawned in
- Went to centcomm
- Spawned the axe
- Turned around to check angles
- Opened the crafting menu
- Crafted it
- Spawned a disciple
- Attacked it 

## Changelog
:cl:
add: Added a makeshift battle axe weapon
add: Adds a makeshift battle axe crafting recipe
imageadd: adds sprites for the makeshift battle axe
imageadd: edits the sprite for the sledgehammer (right hand) to remove stray pixels
/:cl:

### CODE REVIEW

**Comment by DiaFRAME444** [comment]
Menu for the Makeshift Axe seems wrong? Junkblade shows its cutting value of 15, Makeshift Axe doesn't.

**Comment by assortedbeads** [comment]
> Menu for the Makeshift Axe seems wrong? Junkblade shows its cutting value of 15, Makeshift Axe doesn't.

Yeah I had grabbed an old pic accidently, fixed now

**Comment by DiaFRAME444** [comment]
Okay. Uhh, does it have a better wield modifier or something? Seems unusable compared to a junkblade.

**Comment by assortedbeads** [comment]
> Okay. Uhh, does it have a better wield modifier or something? Seems unusable compared to a junkblade.

It doesnt no, it's meant to be a breaching/melee weapon not a straight melee weapon

Junkblade is better in straight fights but cant break down doors/walls.
This should be more leaned towards the latter

**Comment by Humonitarian** [comment]
Does not look makeshift

**Comment by assortedbeads** [comment]
> Does not look makeshift

It looks like the halberd does

**Comment by maikilangiolo** [comment]
> It doesnt no, it's meant to be a breaching/melee weapon not a straight melee weapon

Ah so it's completely useless, because the homewrecker does the same thing except it doesn't do 20 damage (lmao), as well as the new hammers also being infinitely better weapons breaching tools.

File: code/datums/craft/recipes/weapon.dm [hunk 1]
```diff
@@ -197,6 +197,18 @@
 		list(/obj/item/stack/rods, 1)
 	)
 
+
+/datum/craft_recipe/weapon/battleaxe
+	name = "makeshift axe"
+	result = /obj/item/tool/makeshiftaxe
+	steps = list(
+		list(/obj/item/stack/rods, 3), //get three rods for the shaft
```
Comment (vode-code) [inline code/datums/craft/recipes/weapon.dm:205]: if you're going to make it cost 3 steel

File: code/datums/craft/recipes/weapon.dm [hunk 2]
```diff
@@ -197,6 +197,18 @@
...
+		list(QUALITY_WELDING, 10, "time" = 10), //weld them togething into a shaft
+		list(CRAFT_MATERIAL, 3, MATERIAL_PLASTEEL), //get some plasteel
```
Comment (vode-code) [inline code/datums/craft/recipes/weapon.dm:207]: and 3 plasteel

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 3]
```diff
@@ -53,6 +53,32 @@
...
 	tool_qualities = list(QUALITY_CUTTING = 20)
 
+/obj/item/tool/makeshiftaxe
+	name = "makeshift axe"
+	desc = "A heavy plasteel blade affixed to a welded metal shaft, for close up carnage"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "makeshift_axe"
+	item_state = "makeshift_axe"
+	wielded_icon = "makeshift_axe_wielded"
+	matter = list(MATERIAL_STEEL = 2, MATERIAL_PLASTEEL = 1)
```
Comment (vode-code) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:63]: you should make it recycle to 3 steel and plasteel too!

**Review by vode-code — CHANGES_REQUESTED** [review]
materials don't add up.

**Comment by DiaFRAME444** [comment]
> Junkblade is better in straight fights but cant break down doors/walls.

Given the junkblade is a weapon that is just barely passing as a "Usable melee weapon", despite that being all its meant to do, trying to make a weapon that is "Worse at combat in exchange for having a different use" means you've got a item that is utterly horrible at both things.

**Comment by vode-code** [comment]
> Given the junkblade is a weapon that is just barely passing as a "Usable melee weapon", despite that being all its meant to do, trying to make a weapon that is "Worse at combat in exchange for having a different use" means you've got a item that is utterly horrible at both things.

clearly, what we need is a reshuffling of all melee stats, so that the scrap sword is decent *and* the trash axe is, and the real versions are *good*.
I suggest AP reduction and damage increases on all swords.

**Comment by vode-code** [comment]
swords are not the only weapon type, and currently they're balanced as though all swords should have AP, even though it's only plausible for pointy swords.

**Comment by assortedbeads** [comment]
> materials don't add up.

I wanted to imply wasted materials, but I can change it sure

**Comment by assortedbeads** [comment]
> > It doesnt no, it's meant to be a breaching/melee weapon not a straight melee weapon
> 
> Ah so it's completely useless, because the homewrecker does the same thing except it doesn't do 20 damage (lmao), as well as the new hammers also being infinitely better weapons breaching tools.

Homewrecker isnt sharp, is double tac, has worse storage

**Comment by assortedbeads** [comment]
> > Given the junkblade is a weapon that is just barely passing as a "Usable melee weapon", despite that being all its meant to do, trying to make a weapon that is "Worse at combat in exchange for having a different use" means you've got a item that is utterly horrible at both things.
> 
> clearly, what we need is a reshuffling of all melee stats, so that the scrap sword is decent *and* the trash axe is, and the real versions are *good*.
> I suggest AP reduction and damage increases on all swords.

We need swords to have worse ap yes, I didnt think that was in the scope of this pr however, but a future change certainly

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 4]
```diff
@@ -53,6 +53,32 @@
...
+	force = WEAPON_FORCE_DANGEROUS
+	throwforce = WEAPON_FORCE_NORMAL
+	armor_divisor = ARMOR_PEN_MODERATE
+	w_class = ITEM_SIZE_NORMAL
+	slot_flags = SLOT_BACK
+	sharp = TRUE
+	edge = TRUE
+	attack_verb = list("chopped", "torn", "cut", "cleaved", "slashed")
+	hitsound = 'sound/weapons/melee/heavystab.ogg'
+	tool_qualities = list(QUALITY_CUTTING = 20)
```
Comment (Firefox13) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:74]:
```suggestion
	tool_qualities = list(QUALITY_CUTTING = 10)
```
Cutting 20 is kitchen knife level. This is a makeshift axe, not made for cutting bread.

File: code/game/objects/items/weapons/tools/simple_weapons.dm (hunk 3 above)
Comment (assortedbeads) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:63]: > you should make it recycle to 3 steel and plasteel too!

fixed

File: code/game/objects/items/weapons/tools/simple_weapons.dm (hunk 4 above)
Comment (assortedbeads) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:74]: That's fair, didnt consider that

**Comment by Firefox13** [comment]
Sprite looks a little wonky to me - one-handed and wielded look too different (length-wise)

**Comment by assortedbeads** [comment]
> Sprite looks a little wonky to me - one-handed and wielded look too different (length-wise)

It's has the same dimensions as the fireaxe I believe? Would you prefer the hand to be implied to be grasping the back of the blade like this?

Different Dimensions

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 5]
```diff
@@ -53,6 +53,32 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	tool_qualities = list(QUALITY_CUTTING = 20)
 
+/obj/item/tool/makeshiftaxe
+	name = "makeshift axe"
+	desc = "A heavy plasteel blade affixed to a welded metal shaft, for close up carnage"
```
Comment (Firefox13) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:58]:
```suggestion
	desc = "A heavy plasteel blade affixed to a welded metal shaft, for close up carnage."
```

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**